internal/pkg/service/buffer/sink/tablesink/storage/repository/file_repo_test.go

```diff
@@ -1113,15 +1121,60 @@ func TestFileRepository_CloseAllIn(t *testing.T) {
 	require.NoError(t, tokenRepo.Put(sink.SinkKey, keboola.Token{Token: "my-token"}).Do(ctx).Err())
 
 	// Create 2 files, with 2 slices
+	// -----------------------------------------------------------------------------------------------------------------
 	require.NoError(t, fileRepo.Rotate(rb, clk.Now(), sinkKey).Do(ctx).Err())
 	clk.Add(time.Hour)
 	require.NoError(t, fileRepo.Rotate(rb, clk.Now(), sinkKey).Do(ctx).Err())
 
 	// Close the last file
+	// -----------------------------------------------------------------------------------------------------------------
 	clk.Add(time.Hour)
+	etcdLogs.Reset()
 	require.NoError(t, fileRepo.CloseAllIn(clk.Now(), sinkKey).Do(ctx).Err())
+	closeAllInEtcdLogs := etcdLogs.String()
 
 	// Check etcd state
+	// -----------------------------------------------------------------------------------------------------------------
+	etcdlogger.Assert(t, `
+// CloseAllIn - AtomicOp - Read Phase
+➡️  TXN
+  ➡️  THEN:
```

Comment:
Added etcd operations snapshots for:
- File: StateTransition, RotateAllIn, CloseAllIn
- Slice: StateTransition, Rotate, Close

See comments in the snapshot.

These test files are already really big, I will split them in the next PR.

Reply:
After a series of fixes in previous PRs, those transactions now look as intended.